graph-builder/src/registry.rs

```diff
@@ -125,7 +125,16 @@ pub fn fetch_releases(
                 tag.to_owned(),
             )
         })
-        .collect();
+        .filter_map(|release| release)
```

Comment:
Since we aren't actually mapping anything here, I think the following is more correct:

```rust
.filter(|release| release.is_some())
```

I realize using `filter_map` is convenient because the closure returns an Option.

Reply:
`filter_map` also does the mapping from `Option<T>` to `T`, see: https://docs.rs/futures/0.1/futures/stream/trait.Stream.html#method.filter_map

Reply:
Heh, I think I'm getting out of shape. I need to write some more Rust.